dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java

```diff
@@ -11,28 +12,58 @@
 public class Task201014UpdateColumnsValuesInIdentifierTable implements StartupTask {
 
     protected static final String UPDATE_CONTENTLETS =
-            "update identifier set owner=q.i_owner, create_date=q.create_date, asset_subtype=q.asset_subtype from"
-            + "(select cont.identifier, cont.mod_user i_owner, inode.idate create_date, "
-            + "struc.velocity_var_name asset_subtype from contentlet cont, "
-            + "inode inode, structure struc "
-            + "where inode.inode=cont.inode "
-            + "and cont.structure_inode = struc.inode) q where id=q.identifier";
+            "update identifier as ident, "
```

Comment:
Codacy found an issue: [The String literal "update identifier as ident, " appears 5 times in this file; the first occurrence is on line 15](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6379663)